apriltag.c

```diff
@@ -291,16 +291,18 @@ static void quick_decode_init(apriltag_family_t *family, int maxhamming)
     // }
 }
 
-// returns an entry with hamming set to 255 if no decode was found.
 static void quick_decode_codeword(apriltag_family_t *tf, uint64_t rcode,
                                   struct quick_decode_entry *entry)
 {
     struct quick_decode *qd = (struct quick_decode*) tf->impl;
 
+    for (int ridx = 0; ridx < 4; ridx++) {
     // quick_decode might be NULL if detector_add_family_bits() failed
     for (int ridx = 0; qd!=NULL && ridx < 4; ridx++) {
 
         for (int bucket = rcode % qd->nentries;
+             qd->entries[bucket].rcode != UINT64_MAX;
+             bucket = (bucket + 1) % qd->nentries) {
```

Comment:
All those changes are reverting something that you did in an earlier commit "remove formatting changes" and it will be changed in the following "remove formatting changes". Squash this all together logically into the commits that original change this.

Reply:
I have tidied up all the commits into a single commit with (hopefully) clear and minimal changes on the most recent merge.

Hopefully this should make it easier to review.